spec/snapshots/Kitchen_Directions_BakeChapterIntroductions_when_v2_called_on_book_with_chapter_objectives_without_chapter_outline.snap

```diff
@@ -0,0 +1,36 @@
+<body>
+    <div data-type="chapter">
+  <h1 data-type="document-title">Chapter 1 Title</h1>
+  <div class="introduction" data-type="page">
+    
+    <figure class="splash">
+      <div data-type="title">Blood Pressure</div>
+      <figcaption>A proficiency in anatomy and physiology... (credit: Bryan Mason/flickr)</figcaption>
+      <span data-type="media" data-alt="This photo shows a nurse taking a woman&#x2019;s...">
+      <img src="ccc4ed14-6c87-408b-9934-7a0d279d853a/100_Blood_Pressure.jpg" data-media-type="image/jpg" alt="This photo shows a nurse taking..."/>
+      </span>
+    </figure>
+    
+    
+    
+  <div class="intro-body">
+  <div data-type="note" class="chapter-objectives"><h3 class="os-title" data-type="title">
+  <span class="os-title-label">Chapter Objectives</span>
+</h3>
+<div class="os-note-body">
+      
+      <p>After studying this chapter, you will be able to:</p>
+      <ul>
+        <li>Distinguish between anatomy and physiology, and identify several branches of each</li>
+      </ul>
+    </div></div>
+  <div class="intro-text">
+    <h2 data-type="document-title" id="_titlecreatedbycookbook"><span data-type="" itemprop="" class="os-text">Introduction</span>
```

Comment:
@philschatz A minor thing - adding the `id` to intro page in the test cases could be useful, so the test (snapshot) will actually show the id :thinking: